src/constants.ts

```diff
@@ -2,8 +2,8 @@ import { FontFamily } from "./element/types";
 
 export const APP_NAME = "Excalidraw";
 
-export const DRAGGING_THRESHOLD = 10; // 10px
-export const LINE_CONFIRM_THRESHOLD = 10; // 10px
+export const DRAGGING_THRESHOLD = 10; // px
+export const LINE_CONFIRM_THRESHOLD = 8; // px
```

Comment:
```suggestion
export const DRAGGING_THRESHOLD_PX = 10;
export const LINE_CONFIRM_THRESHOLD_PX = 8;
```

What about this, ike we have with `_SEC`..

Reply:
maybe, but there are many more px-constants that we should update as well, then.

Reply:
Maybe in another PR.. or part of #2687 :)